packages/stream-api/yarn.lock

```diff
@@ -598,10 +598,10 @@
     eslint-plugin-prettier "^3.1.0"
     prettier "^1.19.1"
 
-"@vertexvis/frame-streaming-protos@^0.0.16":
-  version "0.0.16"
-  resolved "https://registry.yarnpkg.com/@vertexvis/frame-streaming-protos/-/frame-streaming-protos-0.0.16.tgz#a534347e5923e78bc2463ef1f6c7598462b097a2"
-  integrity sha512-H1kxC7QZ1kfQt7KvnwRo3oYDNrw7okPXwBrtoDY4tvFkwA+Jkb65X6vmDelA1ZJZKEA16FefI245s9Ti7DGLSA==
+"@vertexvis/frame-streaming-protos@^0.0.19":
+  version "0.0.19"
+  resolved "https://npm.ops.vertexvis.io/@vertexvis%2fframe-streaming-protos/-/frame-streaming-protos-0.0.19.tgz#b03b06252a6144d43ccb95e5613eefdeb85bcb83"
```

Comment:
This is causing the build to fail. Most of us probably have `@vertexvis` mapped to our private registry. We should probably add an `.npmrc` file to the root of the project that maps `@vertexvis` to our public NPM repo.

```
@vertexvis:registry=https://registry.npmjs.org/
registry=https://registry.npmjs.org/
```